app/policies/wiki_policy.rb

```diff
@@ -2,11 +2,30 @@ class WikiPolicy < ApplicationPolicy
   
   attr_reader :user, :wiki
```

Comment:
Almost perfect! We need to incorporate the Admin role and the rest of the methods. 

```
class WikiPolicy < ApplicationPolicy
  attr_reader :user, :wiki
  
  def initialize(user, wiki)
    @user = user
    @wiki = wiki
  end
  
  def update?
    user.present? || user.admin?
  end
  
  def new?
    user.present? || user.admin?
  end
  
  def create?
    user.present? || user.admin?
  end
  
  def show?
    user.present? || !user.present?
  end
  
  def edit?
    user.present? || user.admin?
  end
  
  def destroy?
    user.admin?
  end
end
```